templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -57,6 +57,7 @@ Metadata:
           - DeploymentAutomationBranch
           - DeploymentAutomationPlaybook
           - DeploymentAutomationKeyName
+          - CloudWatchIntegration
```

Comment:
Would this parameter perhaps better fit in the "Cluster nodes" parameter group? It isn't really a part of the deployment repository other than it is configured by the ansible and it monitors the application nodes.

Reply:
I wonder if we should move all the deployment settings up to the cluster node section.

Reply:
That makes sense to me.